client/my-sites/plugins/main.jsx

```diff
@@ -402,29 +401,17 @@ export class PluginsMain extends Component {
 				<QueryJetpackPlugins siteIds={ this.props.siteIds } />
 				{ this.renderPageViewTracking() }
 				<SidebarNavigation />
-				<div className="plugins__main">
-					<div className="plugins__header">
-						<FormattedHeader
-							brandFont
-							className="plugins__page-heading"
-							headerText={ this.props.translate( 'Plugins' ) }
-							align="left"
-							subHeaderText={ this.props.translate(
-								'Add new functionality and integrations to your site with plugins. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
```

Comment:
Question: Won't we have this text anymore on the new layout?

Reply:
Good question ;-), can't find the decision right now but we have discussed with @JanaMW27 that we won't have a subheader description any more.